### DISCUSSION TITLE
My github profile

### DISCUSSION THREAD

**Discussion by javierd79** [community/community#33776] [Other Feature Feedback, Questions, & Ideas] [answered]
How's my GitHub profile?

Be completely honest, please

**Comment by librity** [comment, marked as answer]
It looks good!

I would re-frase the following lines:
```markdown
# Welcome to my profile!

I'm an engineering student with passion for both front and back end web development;
A fast learner, I'm always reading about something new.

Feel free to contact me about projects, code questions or just to chat :)
```
It sounds more natural this way.

You can also add badges for each tech you know like I did in my profile:
- https://github.com/librity/librity

Those look really neat.

If you have any online/deployed apps you can add a link to them so people can easily see your work.

Make it your own, add pictures, links, whatever you think represents you.
Also take advantage of GitHub's markdown:
- https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax

**Comment by Sagar-Sharma-7** [comment]
Hello @javierd79 
It's good, but you can still improve.
You can look at my profile to get some ideas.

**Comment by jfullstackdev** [comment]
check mine too, 

https://github.com/jdevstatic
